examples/scroll.html

```diff
@@ -0,0 +1,7 @@
+<html>
```

Comment:
Added this to test that the beacons disappear when they are outside the scroll area and also that tooltips "flip" to the other side if they would be outside the scroll area.